site/index.html

```diff
@@ -110,6 +110,7 @@
                         <ul class="dropdown-menu">
                             <li><a href="">English</a></li>
                             <li><a href="ja/index.html">Japanese</a></li>
+							<li><a href="pt-br/index.html">Portuguese-Brazil</a></li>
```

Comment:
This should change to: Português-Brasil
Also please update the copyright to say 

> and others